how-to-guides/mocha-testnet.md

```diff
@@ -136,19 +136,19 @@ as REST endpoints. This allows for communication with the node using REST
 calls, which can be useful if the client does not support gRPC or HTTP2.
 The default port is 1317.
 
-- [https://api-mocha.pops.one](https://api-mocha.pops.one)
-- [https://api.celestia-mocha.com/](https://api.celestia-mocha.com/)
-- [https://celestia-testnet.brightlystake.com/api](https://celestia-testnet.brightlystake.com/api)
-- [https://api-celestia-mocha.trusted-point.com](https://api-celestia-mocha.trusted-point.com)
-- [https://api-celestia-testnet-01.stakeflow.io/](https://api-celestia-testnet-01.stakeflow.io/)
-- [https://mocha.api.cumulo.me/](https://mocha.api.cumulo.me/)
-- [https://api.archive.mocha.cumulo.com.es](https://api.archive.mocha.cumulo.com.es)
-- [https://api-mocha-full.avril14th.org](https://api-mocha-full.avril14th.org)
-- [https://api-1.testnet.celestia.nodes.guru](https://api-1.testnet.celestia.nodes.guru)
-- [https://api-2.testnet.celestia.nodes.guru](https://api-2.testnet.celestia.nodes.guru)
-- [https://celestia-testnet-api.itrocket.net](https://celestia-testnet-api.itrocket.net)
-- [https://api-celestia-testnet.cryptech.com.ua](https://api-celestia-testnet.cryptech.com.ua)
-- [https://celestia-t-api.noders.services](https://celestia-t-api.noders.services)
+- `https://api-mocha.pops.one`
```

Comment:
`503` for all these for the link checker, and likely readers don't want to click active links anyhow, just copy&pasta them somewhere.

Reply:
a bit less clear how to handle link checks for API endpoints without some digging, likely possible to do with lychee or other tool :thinking:

Reply:
I'm onboard with the suggested change.